.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,35 @@
+name: Build Wheels
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - darshan-util/**
+      - include/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - darshan-util/**
+      - include/*
```

Comment:
I just wanted to make a suggestion here based on something I saw in another project:

> Numpy currently uses `cibuildwheel <https://https://cibuildwheel.readthedocs.io/en/stable/>`_
in order to build wheels through continuous integration services. To save resources, the
cibuildwheel wheel builders are not run by default on every single PR or commit to main.

> If you would like to test that your pull request do not break the wheel builders,
you may either append ``[wheel build]`` to the end of the commit message of the commit...

and then the CI config has something like this:

> .github/workflows/wheels.yml:      contains(needs.get_commit_message.outputs.message, '[wheel build]')

maybe you'll want to defer that to another issue, but wanted to mention that

Reply:
Yeah, I like that idea and planned on asking for feedback for that specifically, because the wheel building process takes a _long_ time (especially for PyPy builds for whatever reason). Building them on demand (maybe on big changes and at release time) makes a of sense. I'll try to address that in this PR once I'm sure things are running properly otherwise. Thanks for the pointer.